pkg/controller/postgresql.go

```diff
@@ -147,12 +147,12 @@ func (d *crdDecoder) Decode() (action watch.EventType, object runtime.Object, er
 
 func (c *Controller) clusterWatchFunc(options metav1.ListOptions) (watch.Interface, error) {
 	options.Watch = true
+	// MIGRATION: FieldsSelectorParam(nil)
```

Comment:
does it mean that the code line was removed during migration cause the method is  not there anymore ?

Reply:
Yes